playground/AspireEventHub/EventHubsApi/Program.cs

```diff
@@ -0,0 +1,26 @@
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Producer;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+
+builder.AddAzureEventHubProducerClient("eventhubns", settings =>
```

Comment:
One of the things that I think makes the whole experience with Aspire better is when we can "deep link" to the resource that we want to use. In the settings callback here you are configuring the name of the hub that you want to set up the producer to communicate with, but I'd want to see the ability to also inject an `EventHubProducerClient` based on purely the data that comes through in a connection string.